lib/box/client.go

```diff
@@ -136,7 +136,7 @@ func dial(socketPath string) (net.Conn, error) {
 	conn, err := net.Dial("unix", socketPath)
 	if err != nil {
 		return nil, checkError(
-			trace.Wrap(err, "failed to connect to planet socket"))
+			trace.Wrap(err, "failed to connect to planet. Check that planet is running. Try `sudo gravity shell`?"))
```

Comment:
This could be more helpful and detect permission issue by accessing the socket file or interpreting the error from `net.Dial` and only suggesting `sudo` in this case.
Also, there's no reason to make user check whether the container is running if the error is a permission issue. Once that's been sorted out, it could fall back to this suggestion if connection still fails.